src/test/resources/features/prodDescription.feature

```diff
@@ -0,0 +1,23 @@
+@ProductDesc
+Feature: Amazon health check
+  
+  @tag1
+  Scenario: Product description for amazon  %$45
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+
+  @tag2 -----@123
```

Comment:
Please check the tagline. Its having "-----@123". Please remove it.

Reply:
I made changes at tagline.